cabal.project

```diff
@@ -100,22 +100,6 @@ source-repository-package
 package cryptonite
   flags: -support_rdrand
 
--- Using a fork until our patches can be merged upstream
-
--- TODO: ADP-1713
-source-repository-package
-  type: git
-  location: https://github.com/biocad/servant-openapi3
-  tag: 4165b837d3a71debd1059c3735460075840000b5
```

Comment:
Reason for it's removal:
* This patch got merged and latest version is picked up by constraint solver.
* It has a bug which leads to issues when being used by `dex-contracts-api` repository, see how `_openApiOpenapi` is defined in [this](https://github.com/biocad/servant-openapi3/blob/10c45b4e302567d7491cd0371317e52bf823b71c/src/Servant/OpenApi/Internal.hs#L230-L240) latest version of code but not in the [earlier](https://github.com/biocad/servant-openapi3/blob/4165b837d3a71debd1059c3735460075840000b5/src/Servant/OpenApi/Internal.hs#L230-L239) one.